app.js

```diff
@@ -34,6 +34,7 @@ program
 	.option('-y, --sync [peers...]', 'peers list to sync with')
 	.option('-l, --log <level>', 'log level')
 	.option('-s, --snapshot <round>', 'verify snapshot')
+	.option('-t, --coverage', 'enable functional tests code coverage')
```

Comment:
yes, only for functional. Unit tests run on their own 'instance' of lisk and they store the coverage straight in the directory: `test/.coverage-unit`. The purpose of using `istanbul-middleware` plugin is to store the code coverage on the test server, which is requested via API calls by the functional tests.